Slim/Player/ProtocolHandlers.pm

```diff
@@ -65,13 +70,18 @@ sub isValidRemoteHandler {
 sub registeredHandlers {
 	my $class = shift;
 
-	return keys %protocolHandlers;
+	return keys %protocolHandlers, keys %URLHandlers;
```

Comment:
When I look how `registeredHandlers()` is being used I'd say the `%URLHandlers` should not be part of that list.